docs/README.md

```diff
@@ -16,6 +16,8 @@ JavaScript and TypeScript [finite state machines](https://en.wikipedia.org/wiki/
 
 New to state machines and statecharts? [Read our introduction](/guides/introduction-to-state-machines-and-statecharts/).
 
+[Download our VSCode extension](https://marketplace.visualstudio.com/items?itemName=mattpocock.xstate-vscode).
```

Comment:
TODO

- [ ] change this link when we deploy the new VS Code extension

Reply:
```suggestion
[Download our VSCode extension](https://marketplace.visualstudio.com/items?itemName=statelyai.stately-vscode).
```